_includes/components/header-and-footer/Footer/index.html

```diff
@@ -0,0 +1,73 @@
+
+<footer class="Footer">
+
+    <div class="u-textAlignCenter u-marginTop20">
+      <a
+      class="Button is-secondary is-inline"
+      target="_blank"
+      href="https://docs.google.com/forms/d/e/1FAIpQLSdTgVXAPqmiT33VF_GvMtEANhxIi3V9BlVrqsHoI0Xy1Bv14w/viewform?entry.1530467677=https%3A%2F%2Fvulekamali.gov.za%2F%2F2018-19%2Ffocus%2F0.html">
```

Comment:
Hardcoding this now is going to be a problem when we do the rest of the migration. I think you should have replaced jekyll `page.url` with something like https://stackoverflow.com/questions/15436407/request-path-in-django-template

Reply:
I don't know why the old code resulted in /focus/0.html but the intention here, as per the old code, is to temlpate in the current URL

```diff
--- a/_includes/components/header-and-footer/Footer/index.html
+++ b/_includes/components/header-and-footer/Footer/index.html
@@ -5,8 +5,7 @@
       <a
       class="Button is-secondary is-inline"
       target="_blank"
-      href="https://docs.google.com/forms/d/e/1FAIpQLSdTgVXAPqmiT33VF_GvMtEANhxIi3V9BlVrqsHoI0Xy1Bv14w/viewform?entry.1530467677={{ 'https://vulekamali.gov.za/' | escape }}{{  page.url | escape }}"
-      >
+      href="https://docs.google.com/forms/d/e/1FAIpQLSdTgVXAPqmiT33VF_GvMtEANhxIi3V9BlVrqsHoI0Xy1Bv14w/viewform?entry.1530467677=https%3A%2F%2Fvulekamali.gov.za%2F%2F2018-19%2Ffocus%2F0.html">
```

Reply:
I see